provider/pkg/gen/dotnet-templates/helm/v2/Chart.cs

```diff
@@ -210,25 +222,366 @@ namespace Pulumi.Kubernetes.Helm.V2
+                        cfgBase = cfg;
+                    }
+                    else
+                    {
+                        var cfg = chartArgs.AsT1;
+                        chart = cfg.Path;
+                        defaultValues = Path.Join(chart, "values.yaml");
+                        cfgBase = cfg;
+                    }
+
+                    // Write overrides file.
+                    var data = JsonSerializer.Serialize(cfgBase.Values);
+                    File.WriteAllText(overrides, data);
+
+                    // Does not require Tiller. From the `helm template` documentation:
+                    //
+                    // >  Render chart templates locally and display the output.
+                    // >
+                    // > This does not require Tiller. However, any values that would normally be
+                    // > looked up or retrieved in-cluster will be faked locally. Additionally, none
+                    // > of the server-side testing of chart validity (e.g. whether an API is supported)
+                    // > is done.
+                    var flags = new List<string>(new[]
+                    {
+                        "template", chart,
+                        "--name-template", releaseName,
+                        "--values", defaultValues,
+                        "--values", overrides
+                    });
+                    if (cfgBase.ApiVersions.Length > 0)
+                    {
+                        flags.Add("--api-versions");
+                        flags.Add(string.Join(",", cfgBase.ApiVersions));
+                    }
+
+                    if (!string.IsNullOrEmpty(cfgBase.Namespace))
+                    {
+                        flags.Add("--namespace");
+                        flags.Add(cfgBase.Namespace);
+                    }
+
+                    if (IsHelmV3())
+                    {
+                        flags.Add("--include-crds");
+                    }
+
+                    var yaml = ExecuteCommand("helm", flags.ToArray(), new Dictionary<string, string>());
+                    return ParseTemplate(
+                        yaml, cfgBase.Transformations, cfgBase.ResourcePrefix, dependencies, cfgBase.Namespace);
+                }
+                catch (Exception e)
+                {
+                    // Shed stack trace, only emit the error.
+                    throw new ResourceException(e.Message, this);
+                }
+                finally
+                {
+                    chartDirectory.Delete(true);
+                }
+            });
+            RegisterResources(resources);
+        }
+        private static string GetName(Union<ChartArgs, LocalChartArgs> config, string releaseName)
+        {
+            var prefix = config.Match(v => v.ResourcePrefix, v => v.ResourcePrefix);
+            return string.IsNullOrEmpty(prefix) ? releaseName : $"{prefix}-{releaseName}";
+        }
+
+        private static bool IsHelmV3()
```

Comment:
Is this still needed? Is it still possible to use Chart V2 for Helm 3?

Reply:
Yes, the v2 SDK supports either CLI.